### PR TITLE
Improve travis script

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -124,70 +124,35 @@ script:
...
 - export DEPLOY_FILE="";
 - if [ -z $TEST ]; then
+    ls /tmp/pingviewer*;
     wget -c https://github.com/probonopd/uploadtool/raw/master/upload.sh;
   fi
 - if [ ! -z $TRAVIS_BRANCH ] && [ "$TRAVIS_BRANCH" != "master" ] ; then
     export UPLOADTOOL_SUFFIX=$TRAVIS_BRANCH;
   fi
 - if [[ "${TRAVIS_OS_NAME}" = "linux" && -z $TEST ]]; then
     bash ./upload.sh ./pingviewer*.AppImage;
```
Comment (jaxxzer) [inline .travis.yml]: `./pingviewer*.AppImage` -> `/tmp/pingviewer*.AppImage`

better yet, move DEPLOY_FILE above this and use that. Same as below?

File: .travis.yml [hunk 2]
```diff
@@ -124,70 +124,35 @@ script:
...
-    mv pingviewer.dmg pingviewer-${QT_CONFIG}.dmg;
+    ./tools/compile.sh || travis_terminate 1;
   fi
 
 after_success:
 # Upload files in GitHub
 - export DEPLOY_FILE="";
 - if [ -z $TEST ]; then
+    ls /tmp/pingviewer*;
     wget -c https://github.com/probonopd/uploadtool/raw/master/upload.sh;
```
Comment (jaxxzer) [inline .travis.yml]: Should we fork this? What if he changes it and breaks our build?

Comment (patrickelectric) [reply .travis.yml]: For small projects such this one I would say yes. But probonopd and theassassin are pretty popular in deployment tools, I trusty them. Besides that, linuxdeployqt and uploadtool both are tested and deployed by it's own release, so if they do not work, they will not be available to download :)

File: .travis.yml (hunk 1 above)
Comment (patrickelectric) [reply .travis.yml]: Done.

**Pull request merged**